File: VB/UserDiffsToDBVB/UserDiffsToDB.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.BaseImpl

Namespace UserDiffsToDB.Module
    Public Class Updater
        Inherits ModuleUpdater
        Public Sub New(ByVal objectSpace As ObjectSpace, ByVal currentDBVersion As Version)
            MyBase.New(objectSpace, currentDBVersion)
        End Sub
        Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
            MyBase.UpdateDatabaseAfterUpdateSchema()

            Dim adminUser As SimpleUser = ObjectSpace.FindObject(Of SimpleUser)(New BinaryOperator("UserName", "Sam"))
            If adminUser Is Nothing Then
                adminUser = ObjectSpace.CreateObject(Of SimpleUser)()
                adminUser.UserName = "Sam"
                adminUser.FullName = "Sam"
            End If
            adminUser.IsAdministrator = True
            adminUser.SetPassword("")
            adminUser.Save()

            Dim user As SimpleUser = ObjectSpace.FindObject(Of SimpleUser)(New BinaryOperator("UserName", "John"))
            If user Is Nothing Then
                user = ObjectSpace.CreateObject(Of SimpleUser)()
                user.UserName = "John"
                user.FullName = "John"
            End If
            user.IsAdministrator = False
            user.SetPassword("")
            user.Save()
        End Sub
    End Class
End Namespace

File: CS/UserDiffsToDB/UserDiffsToDB.Module/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.ExpressApp;
using DevExpress.Persistent.BaseImpl;

namespace UserDiffsToDB.Module {
    public class Updater : ModuleUpdater {
        public Updater(ObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();

            SimpleUser adminUser = ObjectSpace.FindObject<SimpleUser>(new BinaryOperator("UserName", "Sam"));
            if (adminUser == null) {
                adminUser = ObjectSpace.CreateObject<SimpleUser>();
                adminUser.UserName = "Sam";
                adminUser.FullName = "Sam";
            }
            adminUser.IsAdministrator = true;
            adminUser.SetPassword("");
            adminUser.Save();

            SimpleUser user = ObjectSpace.FindObject<SimpleUser>(new BinaryOperator("UserName", "John"));
            if (user == null) {
                user = ObjectSpace.CreateObject<SimpleUser>();
                user.UserName = "John";
                user.FullName = "John";
            }
            user.IsAdministrator = false;
            user.SetPassword("");
            user.Save();
        }
    }
}